pages.zh/osx/pbcopy.md

```diff
@@ -1,11 +1,12 @@
 # pbcopy
 
-> 将标准输出放入剪贴板（命令行里的 cmd + C）。
+> 将来自标准输入的数据放入剪贴板。
+> 相当于在键盘上按下 Cmd + C。
```

Comment:
```suggestion
> 相当于在键盘上按下 Cmd + C.
```
Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#chinese-specific-rules _(5. Use a half-width punctuation to end a sentence when the last character is half-width.)_